pkg/systemd/services/services.jsx

```diff
@@ -33,7 +33,6 @@ import {
     ToolbarContent,
     ToolbarItem,
```

Comment:
There are two commits that "move create timer button". Can we squash those?

Reply:
Done